### PR TITLE
Add queries, alter table  and insert data.

### PR DESCRIPTION
**Pull request by NelsonAraujo93**
**Hello reviewer**,

In this pull request i followed the next stapes:
## Alter Table animals ##
- Add column species to table animals. (image 1)

add species column
Image 1.

## Insert data into animals Table ##
- Insert multiple rows into animals table. (image 2)

new data
Image 2.

## Transactions ##

1. I did many transactions, the first one consist by updating species to unspecified and then rollback (Images 3-4)

update species to unespecified
Image 3.

after rollback
Image 4.

After that inside another transaction i update all animals species with mon in their names to digimon (image 5) and then all null species to pokemon (image 6). Finally i verified and commit.(image 7)

update all mon names
Image 5.

update all pokemons from null
Image 6.

Verified commit data
Image 7.

2. The second transaction was about to delete all the rows and then rollback.(images 8-9)

Delete all from animals
Image 8.

Rollback delete transaction
Image 9.

3. The last transaction started with a delete and savepoint (image 10). Then update all animal weight multiplying by -1 (image 11) and perform a rolling back to savepoint (image 12). Finally updated all animals with negative weights multiplying by -1 (image 13) and commit;

Delete animals burn after 2022-01-01
Image 10.

update all weights  switch between positive and negative
Image 11.

Rollback to SP1
Image 12.

Update all negatives weights to positive
Image 13.

## Queries ##

Next i performed all the queries asked for the activity

- [x] How many animals are there?

Count animals

- [x] How many animals have never tried to escape?

Animals who never tried escape

- [x] What is the average weight of animals?

Avg weight

- [x] Who escapes the most, neutered or not neutered animals?

who escaped more neutered or nonNeutered

- [x] What is the minimum and maximum weight of each type of animal?

max min weight from species

- [x] What is the average number of escape attempts per animal type of those born between 1990 and 2000?

escape attemps grouped by especies burn between 1990 and 2000

**Thanks for reviewing my code, greeting Nelson** 😄.

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -14,4 +14,72 @@ SELECT * from animals WHERE neutered = true;
...
-SELECT * from animals WHERE weight_kg BETWEEN 10.4 and 17.3;
\ No newline at end of file
+SELECT * from animals WHERE weight_kg BETWEEN 10.4 and 17.3;
+
+/* Update inside a transaction and rollback*/
+Begin;
+
+UPDATE animals
+set species = 'unespecified';
+
```
Comment (addod19) [inline queries.sql:24]: - [x] It would be better if we verify that the change was made after setting species to unspecified

File: queries.sql [hunk 2]
```diff
@@ -14,4 +14,72 @@ SELECT * from animals WHERE neutered = true;
...
+Rollback;
+
```
Comment (addod19) [inline queries.sql:26]: - [x] It would be better if we verify that the change was undone

File: queries.sql [hunk 3]
```diff
@@ -14,4 +14,72 @@ SELECT * from animals WHERE neutered = true;
...
+/* Update inside a transaction and commit*/
+Begin;
+
+UPDATE animals
+set species = 'digimon'
+where name like '%mon%';
```
Comment (addod19) [inline queries.sql]: - [x] It would be better if we fix our query by changing `%mon%` to `%mon`

File: queries.sql [hunk 4]
```diff
@@ -14,4 +14,72 @@ SELECT * from animals WHERE neutered = true;
...
+
+DELETE from animals;
+
+Rollback;
+
+/*Update insde transaction with savepoint, rollback and commit*/
+Begin;
+
+Delete from animals
+where date_of_birth >= '2022-01-01';
```
Comment (addod19) [inline queries.sql]: - [x] Let us kindly use `>` instead of `>=` since we are deleting records after Jan 1st, 2022

**Review by addod19 — CHANGES_REQUESTED** [review]
### Hi @NelsonAraujo93, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights


## Required Changes ♻️
_Check the comments under the review._

- [x] It would be better if we make our README file descriptive and professional by fixing the following:
1. Broken MIT lincense link
2. Broken deployed link, kindly remove it since the app is not deployed
3. Also the key features and future feature needs proper text instead of the feature_1_2_3 we have currently
4. Since this is not a collaborative project, let us fix the author section to reflect the current project and let us use real social media handle because recruiters will use it to contact us
5. The header and description should be fixed


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (danmainah) [reply queries.sql:24]: - [ ] @NelsonAraujo93 as previously requested kindly add a query to verify the change was made after setting species to unspecified.

File: queries.sql (hunk 2 above)
Comment (danmainah) [reply queries.sql:26]: - [ ] @NelsonAraujo93 as previously requested kindly add a query to verify the change was made after the rollback.

File: README.md [hunk 5]
```diff
@@ -63,33 +31,18 @@ After you're finished please remove all the comments and instructions!
...
 <!-- PROJECT DESCRIPTION -->
 
-# 📖 [your_project_name] <a name="about-project"></a>
+# 📖 [Vet_Clinic] <a name="about-project"></a>
 
-> Describe your project in 1 or 2 sentences.
+This project is created to practice and test our knowledge in SQL and to learn best practices of it.
 
-**[your_project__name]** is a...
+**[Vet_Clinic]** is a SQL database project focused on practice and learn SQL basics.
```
Comment (danmainah) [inline README.md]: - [x] @NelsonAraujo93 great work implementing the changes requested however you have added two descriptions of the project kindly remove one.

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @NelsonAraujo93 , 

Great work on making the changes requested by a previous reviewer 👏  👏 👏
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.


 ## Highlights
✔️ Gitflow was correctly used

 # Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by BregornOriginal — APPROVED** [review]
## Hi @NelsonAraujo93 

## Your project is complete!  There is nothing else to say other than... it's time to merge it 

## Congratulations! 

giphy

## Highlights

✅ Good work making changes requested by the previous reviewer
✅ Good commit history

## Optional suggestions
Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.
Cheers and Happy coding!:aplauso::aplauso::aplauso:
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

Please, remember to tag me in your question so I can receive the notification.
------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)[.](https://airtable.com/shrQAqnBwek5a0O0s)._)

**Pull request merged**